engine/runtime-parser-processor/src/main/java/org/enso/runtime/parser/processor/field/Field.java

```diff
@@ -15,7 +15,7 @@
 public abstract class Field {
   protected final TypeMirror type;
   protected final String name;
-  private final ProcessingEnvironment procEnv;
+  protected final ProcessingEnvironment procEnv;
```

Comment:
```suggestion
  final ProcessingEnvironment procEnv;
```
why it should be protected?